src/app/screens/legalLinks/index.tsx

```diff
@@ -64,14 +64,20 @@ const SwitchContainer = styled.div((props) => ({
   fontSize: '0.875rem',
 }));
 
+const DataCollectionDescription = styled.div((props) => ({
+  ...props.theme.body_m,
+  color: props.theme.colors.white['200'],
```

Comment:
```suggestion
  color: props.theme.colors.white_200,
```

could we stop using the white['200'] pattern please because it breaks type safety